scenarios/web/small/core/2.1/mvc/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,74 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>@ViewData["Title"] - TemplateTest2._1</title>
```

Comment:
Replace `TemplateTest2._1` with `core` for consistency with 2.0 app.

Reply:
Oops. Will do